index.js

```diff
@@ -234,13 +237,27 @@ function emitDrain (sonic) {
 
 inherits(SonicBoom, EventEmitter)
 
-SonicBoom.prototype.write = function (data) {
+function mergeBuf (bufs, len) {
+  if (bufs.length === 0) {
+    return kEmptyBuffer
+  }
+
+  if (bufs.length === 1) {
+    return bufs[0]
+  }
+
+  return Buffer.concat(bufs, len)
```

Comment:
This is the source of the slowdown. We should not merge them, but rather keep them as a list.

Reply:
I've tried having a list here, but tests that expect single flush instead of multiples break. Will work on it further now that there are 2 separate content modes

Reply:
tried fs.writev to avoid concat, but that made it slower than writeStream